### ISSUE TITLE
Test on Ubuntu 20.04

### ISSUE THREAD

**Issue by Asphaltt** [gojue/ecapture#693]
It failed to run on Ubuntu 20.04:

```log
couldn't start bootstrap manager error:2 errors occurred:
	* error:opening Kprobe: creating perf_kprobe PMU (arch-specific fallback for "__x64_sys___sys_connect_file"): token __x64___x64_sys___sys_connect_file: not found: no such file or directory, funcName:__x64_sys___sys_connect_file, isRet:false, section:kprobe/__sys_connect_file, {UID:kprobe_sys_connect_file, EbpfFuncName:probe_connect_file}
	* error:opening Kprobe: creating perf_kprobe PMU (arch-specific fallback for "__x64_sys_do_accept"): token __x64___x64_sys_do_accept: not found: no such file or directory, funcName:__x64_sys_do_accept, isRet:true, section:kretprobe/do_accept, {UID:kretprobe_do_accept, EbpfFuncName:retprobe_do_accept}

, probes activation validation failed .
```

I'll fix it later.

**Comment by Asphaltt** [comment]
> remove prefix "_sys", ebpfmanager will automatically search for the prefix of kprobe from the current kernel configuration.

Nope.

It's because `__sys_connect_file` and `do_accept` are not found on v5.4 kernel.

**Comment by Asphaltt** [comment]
> Same error occurs on kernel 4.19

I'll fix it asap! Then test it on v4.19 kernel.
